### PR TITLE
#9878 disable 'comment' field in type 'interest'

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

**Review by jayay — CHANGES_REQUESTED** [review]
There's an easier and cleaner way. Did you happen to take a look at `\onOffice\WPlugin\Field\Collection\FieldsCollectionConfiguratorForm::configureForInterestForm`?

**Comment by hungnc89** [comment]
> There's an easier and cleaner way. Did you happen to take a look at `\onOffice\WPlugin\Field\Collection\FieldsCollectionConfiguratorForm::configureForInterestForm`?

Thanks for helping me, I reverted & updated code in `\onOffice\WPlugin\Field\Collection\FieldsCollectionConfiguratorForm::configureForInterestForm`, thanks

File: tests/TestClassFieldsCollectionConfiguratorForm.php [hunk 1]
```diff
@@ -74,6 +74,27 @@ public function testConfigureForInterestForm()
...
+		$pSubject = new FieldsCollectionConfiguratorForm;
+		$pFieldsCollectionNew = $pSubject->configureForInterestForm($pFieldsCollection);
+		$this->assertCount(1, $pFieldsCollectionNew->getAllFields());
+		foreach ($pFieldsCollectionNew->getAllFields() as $pField) {
+			$pFieldOriginal = $pFieldsCollection->getFieldByModuleAndName
+			($pField->getModule(), $pField->getName());
+			$this->assertNotSame($pFieldOriginal, $pField);
+			$this->assertThat($pField, $this->callback(function(Field $pField) {
+				return
+					($pField->getName() === 'objekttyp' && $pField->getType() === FieldTypes::FIELD_TYPE_MULTISELECT) ||
```
Comment (jayay) [inline tests/TestClassFieldsCollectionConfiguratorForm.php:92]: I'm missing the check for `$pField->getName() !== 'krit_bemerkung_oeffentlich'` here. Other than that it looks pretty good

**Review by jayay — CHANGES_REQUESTED** [review]
If you could add that one assert that'd be good. Other than that, I think it's good.

File: tests/TestClassFieldsCollectionConfiguratorForm.php (hunk 1 above)
Comment (hungnc89) [reply tests/TestClassFieldsCollectionConfiguratorForm.php:92]: Ok, I updated code, please check help me, thanks

**Review by jayay — APPROVED** [review]

**Review by jayay — APPROVED** [review]

**Comment by jayay** [comment]
As discussed, Applicant Search Form should have been affected.

**Pull request closed without merging**